src/main/java/com/sparkplatform/api/SparkSession.java

```diff
@@ -99,6 +99,7 @@ public int getExpiresIn()
 	public void setExpiresIn(int expiresIn)
 	{
 		this.expiresIn = expiresIn;
+		this.refreshTimeout = expiresIn / 2;
```

Comment:
Was this intentional? I think it's reasonable to do this if the `refreshTimeout` isn't set yet, but it could override a previously set value (and could be confusing).